src/hardware/configuration.rs

```diff
@@ -439,6 +440,12 @@ pub fn setup(
         )
     };
 
+    let digital_inputs = {
+        let di0 = gpiog.pg9.into_floating_input();
```

Comment:
That's somewhat unfortunate that we don't have any pull on the digital inputs - leaving them floating doesn't feel right, but we can't do much after the input buffer stage.

Reply:
https://github.com/sinara-hw/Stabilizer/issues/91